### ISSUE TITLE
Uncaught Error: Parse Error

### ISSUE THREAD

**Issue by ghost** [leny/atom-w3c-validation#4]
When I saved the HTML file, and it loads the plugin, it shows an error.

---

**Atom Version**: 0.188.0
**System**: linux 3.19.3-200.fc21.x86_64
**Thrown From**: Atom Core
### Stack Trace

Uncaught Error: Parse Error

```
At events.js:94

Error: Parse Error
    at Error (native)
    at Socket.socketOnData (_http_client.js:311:20)
    at Socket.emit (events.js:116:17)
    at readableAddChunk (_stream_readable.js:164:16)
    at Socket.Readable.push (_stream_readable.js:127:10)
    at TCP.onread (net.js:530:20)
```
### Commands

```
     -0:05.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:04.8 core:copy (atom-text-editor.editor.is-focused)
     -0:04.5 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:04.5 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:04.5 editor:newline (atom-text-editor.editor.is-focused)
     -0:04.4 core:paste (atom-text-editor.editor.is-focused)
  9x -0:04.2 core:move-left (atom-text-editor.editor.is-focused)
     -0:02.8 core:backspace (atom-text-editor.editor.is-focused)
     -0:02.1 core:save (atom-text-editor.editor.is-focused)
     -0:01.9 find-and-replace:select-next (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {},
  "editor": {
    "invisibles": {},
    "fontSize": 14,
    "tabLength": 4
  }
}
```
### Installed Packages

``` coffee
# User
emmet, v2.3.6
php-getters-setters, v0.3.0
w3c-validation, v0.1.3

# Dev
No dev packages
```

**Comment by leny** [comment]
Hi, sorry for the long delay. I can't reproduce the error since some corrections made to fix the API v1.0 deprecation warnings.

Seems to be fixed in 0.2.0. Thanks for the report.

**Comment by ghost** [comment]
OK. Thank you for answer! :)
